### PR TITLE
Creation of Works and About

### PR DESCRIPTION
**Pull request by mikemtzp**
## Creation of Works and About
For this feature I:

- Eliminated the phone status bar
- Add the works and about HTML and CSS sections.
- Fixed the general HTML structure
- Used both Flexbox and Grid to build the project sections.
- Updated the readme.md

### CODE REVIEW

**Comment by mikemtzp** [comment]
> ## Creation of Works and About
> For this feature I:
> 
> * Eliminated the phone status bar
> * Add the works and about HTML and CSS sections.
> * Fixed the general HTML structure
> * Used both Flexbox and Grid to build the project sections.
> * Updated the readme.md

To the code reviewer:

Hope you're doing fine! If the project needs any corrections could you please explain to me in great detail which parts of the template are needed to be built either with Flexbox or Grid?

Because it is not very clear what is a "card" or "individual card" and which should be the "main container"
image


I used Grid to adjust the whole .works section and also to accommodate the programming languages in the project part.

image

Let me know in detail, with screenshots of the sections, if changes are needed!
Thank you very much! 😁

File: index.html [hunk 1]
```diff
@@ -58,9 +47,158 @@ <h1>Hey There. <br>I'm Mike</h1>
...
+
+      <h2 class="about-title">About me</h2>
+      <p class="about-text">
+        Hello I'm a software developer! I can help you build a product,
+        feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded,
+        don't hestiate to contact me.
+      </p>
+
+      <p class="w1-see">Get My Resume</p>
```
Comment (Dipeshtwis) [inline index.html]: - [ ] The button (e.g., "Get my resume") should be a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122))

File: index.html [hunk 2]
```diff
@@ -58,9 +47,158 @@ <h1>Hey There. <br>I'm Mike</h1>
...
+          or sign-ups required. has been the industry's standard
+        </p>
+
+        <ul class="project-lan">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+
+        <p class="project-see">See Project</p>
```
Comment (Dipeshtwis) [inline index.html]: - [ ] The button (e.g., "See project") should be a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122))

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Set up a "Hello Microverse".
+# Building My Portfolio.
 > In this project, I will imitate a header from a Figma Template. My goal here is to practice the use of flexbox and my HTML and CSS abilities to build similar sections of a webpage. No complex coding is required for this exercise.
```
Comment (Dipeshtwis) [inline README.md:3]: - [ ] I would recommend adding the proper steps to clone and run the project locally. You can use the below hint to understand.

Hint:
```
$ git clone https://github.com/mikemtzp/my-portfolio.git
$ cd my-portfolio
$ open index.html file
```

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mikemtzp, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:+1: The portfolio is looking good.
:+1: Linters are passing. :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mikemtzp** [comment]
Changes made! @Dipeshtwis

**Review by Emmyn5600 — APPROVED** [review]
Hi @mikemtzp 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] N/A

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**